Makefile

```diff
@@ -29,7 +29,7 @@ venv: $(VENV)/bin/activate ## Create and activate virtual environment
 $(VENV)/bin/activate: sdk/python/setup.py
 	@echo "VENV=$(VENV)"
 	@test -d $(VENV) || python3 -m venv $(VENV)
-	pip install -e sdk/python
+	@$(VENV)/bin/pip show kfp-tekton >/dev/null 2>&1 || $(VENV)/bin/pip install -e sdk/python
```

Comment:
- check if `kfp-tekton` is already installed (in Travis/CI context it will have been done during `install`)
- use the specified `VENV` as opposed to any `pip` availlable by the shell

Reply:
Maybe add the above text in comments?

Reply:
@ckadner do you want to add the above comments in the Makefile?